doc/experiment.rst

```diff
@@ -12,316 +38,211 @@ Jupyter notebook, interactive python session, or Python file and use the
+(e.g., Slurm) and the available compute resources on the system.
+Optionally, the user may also specify a BatchSettings object if
+the model should be launched as a batch on the WLM system.
 
 Ensemble
-========
-
-In addition to a single model, SmartSim has the ability to launch an
+^^^^^^^^
+In addition to a single model, SmartSim has the ability to launch a
 ``Ensemble`` of ``Model`` applications simultaneously.
-
-An ``Ensemble`` can be constructed in three ways:
-  1. Parameter expansion (by specifying ``params`` and ``perm_strat`` argument)
-  2. Replica creation (by specifying ``replicas`` argument)
-  3. Manually (by adding created ``Model`` objects) if launching as a batch job
-
 Ensembles can be given parameters and permutation strategies that define how the
-``Ensemble`` will create the underlying model objects.
-
-Three strategies are built in:
-  1. ``all_perm``: for generating all permutations of model parameters
-  2. ``step``: for creating one set of parameters for each element in `n` arrays
-  3. ``random``: for random selection from predefined parameter spaces
-
-Here is an example that uses the ``random`` strategy to intialize four models
-with random parameters within a set range. We use the ``params_as_args`` field
-to specify that the randomly selected learning rate parameter should be passed
-to the created models as a executable argument.
-
-.. code-block:: bash
-
-  import numpy as np
-  from smartsim import Experiment
-
-  exp = Experiment("Training-Run", launcher="auto")
-
-  # setup ensemble parameter space
-  learning_rate = list(np.linspace(.01, .5))
-  train_params = {"LR": learning_rate}
-
-  # define how each member should run
-  run = exp.create_run_settings(exe="python",
-                                exe_args="./train-model.py")
-
-  ensemble = exp.create_ensemble("Training-Ensemble",
-                                params=train_params,
-                                params_as_args=["LR"],
-                                run_settings=run,
-                                perm_strategy="random",
-                                n_models=4)
-  exp.start(ensemble, summary=True)
-
-
-A callable function can also be supplied for custom permutation strategies.  The
-function should take two arguments: a list of parameter names, and a list of
-lists of potential parameter values. The function should return a list of
-dictionaries that will be supplied as model parameters. The length of the list
-returned will determine how many ``Model`` instances are created.
-
-For example, the following is the built-in strategy ``all_perm``:
+``Ensemble`` will create the underlying model objects. An ensemble is created
+with the function ``Experiment.create_ensemble()`` and requires
+one of the subsequent sets of arguments upon initialization:
+1. run_settings and params
+2. run_settings and replicas
+3. batch_settings
+4. batch_settings, run_settings, and params
+5. batch_settings, run_settings, and replicas
+
+RunSettings
```

Comment:
As discussed, I don't think RunSettings and BatchSettings should be in this section.